CONTRIBUTING.md

```diff
@@ -1,129 +1,139 @@
 
-## Attribution
+<!-- omit in toc -->
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/MeikelLP/quantum-core-x/issues).
+
+- Use a **clear and descriptive title** for the issue to identify the suggestion.
+- Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+- **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
+- You may want to **include screenshots and animated GIFs** which help you demonstrate the steps or point out the part which the suggestion is related to. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux. <!-- this should only be included if the project has a GUI -->
+- **Explain why this enhancement would be useful** to most Quantum Core X users. You may also want to point out the other projects that solved it better and which could serve as inspiration.
+
+<!-- You might want to create an issue template for enhancement suggestions that can be used as a guide and that defines the structure of the information to be included. If you do so, reference it here in the description. -->
+
+### Your First Code Contribution
+
+Please refer to the [docs](docs/docs/Getting%20Started/developer.md).
 
-This Code of Conduct is adapted from the [Contributor Covenant][homepage],
-version 2.0, available at
-https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+### Improving The Documentation
 
-Community Impact Guidelines were inspired by [Mozilla's code of conduct
-enforcement ladder](https://github.com/mozilla/diversity).
+Adjusting files in `docs/` will trigger a rebuild of the documentation page on [https://meikellp.github.io/quantum-core-x/](https://meikellp.github.io/quantum-core-x/).
 
-[homepage]: https://www.contributor-covenant.org
+## Styleguides
 
-For answers to common questions about this code of conduct, see the FAQ at
-https://www.contributor-covenant.org/faq. Translations are available at
-https://www.contributor-covenant.org/translations.
+Refer to [C# identifier names](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/identifier-names) as the default style guide. It is the default in Visual Studio and Rider.
+Any override will be enforced by tools (.editorconfig or similar) from us - so don't worry.
+
+### Commit Messages
+
+We are using [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
```

Comment:
Are we enforcing this? Maybe add an explicit line if so, e.g. `Pull Requests and commits that do not follow this standard will not be reviewed and instead immediately rejected.`

Reply:
Sounds too harsh for me. Contributions should be welcomed not restricted. We can always squash and adjust commit messages 👍

Reply:
Oh, yeah, good point!